application.js

```diff
@@ -1,16 +1,25 @@
 var mbaasApi = require('fh-mbaas-api');
 var express = require('express');
 var mbaasExpress = mbaasApi.mbaasExpress();
-var cors = require('cors');
+//var cors = require('cors');
+var proxy = require('express-http-proxy');
 
 // list the endpoints which you want to make securable here
 var securableEndpoints;
 securableEndpoints = [];
 
 var app = express();
 
+var jbpmPort = process.env.JBPM_PORT || process.env.OPENSHIFT_JBPM_PORT || 8080;
+var jbpmHost = process.env.JBPM_IP || process.env.OPENSHIFT_JBPM_IP || '0.0.0.0';
+var jbpmUsername = process.env.JBPM_USERNAME || process.env.OPENSHIFT_JBPM_USERNAME || null;
+var jbpmPassword = process.env.JBPM_PASSWORD || process.env.OPENSHIFT_JBPM_PASSWORD || null;
+
+var port = process.env.FH_PORT || process.env.OPENSHIFT_NODEJS_PORT || 8001;
+var host = process.env.OPENSHIFT_NODEJS_IP || '0.0.0.0';
+
 // Enable CORS for all requests
-app.use(cors());
+//app.use(cors());
```

Comment:
might as well remove this line altogether (and the `// Enable CORS for all requests` comment)